content/en/docs/best-practices/common.md

```diff
@@ -37,8 +37,9 @@ In the [CSV](/docs/concepts/crds/clusterserviceversion) manifest of your operato
 
 It is recommended you provide this information. Otherwise, it would mean that your operator project can be distributed and installed in any cluster version available, which is not necessarily the case for all projects.
 
-[operator-best-practices]: https://operator-framework.github.io/community-operators/best-practices/
+[operator-best-practices]: https://master.sdk.operatorframework.io/docs/best-practices/best-practices/
```

Comment:
Is it possible to pin this to a particular releases rather than master?

Reply:
Does OLM have any sense of a SDK compatibility matrix? Generally agree that parking on a specific release is better than master given links can break over time.

Reply:
@anik120 we have the latest one which is usually the link used. 
However, this doc is not present there yet. So, I added a todo for we know that we need to update that after the next release.